casee.go

```diff
@@ -130,6 +130,35 @@ func IsFlatCase(s string) bool {
 	}
 }
 
+// Convert argument to UPPER_CASE style string.
+// If argument is empty, return itself.
+func ToUpperCase(s string) string {
```

Comment:
I don't think *UpperCase* usually means combined with underscores. So wouldn't `ToUpperSnake` be more appropriate?

Reply:
There are different opinions on this. https://stackoverflow.com/questions/17326185/what-are-the-different-kinds-of-cases and https://en.wikipedia.org/wiki/Naming_convention_(programming)#Multiple-word_identifiers
Upper case *without* underscores seems to be consistently called "upper flat case".
I don't care much about name for UPPER_CASE - would take whatever name you would suggest.